src/bun.js/webcore/blob.zig

```diff
@@ -1583,7 +1584,7 @@ pub const Blob = struct {
         data: Data,
 
         mime_type: MimeType = MimeType.none,
-        ref_count: u32 = 0,
+        ref_count: std.atomic.Value(u32) = std.atomic.Value(u32).init(0),
```

Comment:
we should remove the default value or change it to `1` because of the assertion in `ref()`